src/java.base/share/classes/java/io/Console.java

```diff
@@ -580,7 +580,8 @@ public void flush() {
      * the {@code Console}.
      * <p>
      * The returned charset corresponds to the input and output source
-     * (e.g., keyboard and/or display) specified by the host environment or user.
+     * (e.g., keyboard and/or display) specified by the host environment or user,
+     * which defaults to the one based on {@link System##stdout.encoding stdtout.encoding}.
```

Comment:
In System.getProperties, this is specified for as the encoding for System.out. That may need to be expanded to say Console too.

Reply:
Added `System.console()` mention to `stdout.encoding`. Also removed `Console` mention from `System.out`, as it no longer provides extra meaning.